Tips-for-common-editors.md

```diff
@@ -0,0 +1,141 @@
+
+   ![Sublime with a file open for editing](images/commonEditors/sublimeEditor.png)
+
+3. To edit the file, just start typing!
+
+Tips:
+
+* [Package Control](http://www.sublimelinter.com/en/stable/) is the *de facto* package manager for Sublime. You can install it by selecting "Install Package Control" from the "Tools" menu. Then you can install packages by opening the command pallete (CMD-SHIFT-P on MacOS or CTRL-SHIFT-P on Windows) and typing `Package Control: Install Package`. We recommend the following packages:
+
+  * [`SublimeLinter`](https://packagecontrol.io/packages/SublimeLinter) is a base linter package. To add support for pylint and eslint, install the following packages as well:
+
+    * [`SublimeLinter-eslint`](https://packagecontrol.io/packages/SublimeLinter-eslint)
+
+    * [`SublimeLinter-pylint`](https://packagecontrol.io/packages/SublimeLinter-pylint)
+
+    To configure them, open the SublimeLinter settings from the menu: Preferences > Package Settings > SublimeLinter > Settings. Add these lines to the file that opens (should be mostly blank to start), replacing `PARDIR` with the absolute path to the folder containing both the `oppia_tools` and `oppia` directories:
+
+    ```json
+    {
+        "linters": {
+            "pylint": {
+                "executable": "PARDIR/oppia_tools/pylint-1.9.5/bin/pylint",
+                "env": {
+                    "PYTHONPATH": "PARDIR/oppia_tools/pylint-1.9.5/:PARDIR/oppia_tools/pylint-quotes-0.1.8/"
+                }
+            },
+            "eslint": {
+                "executable": "PARDIR/oppia/node_modules/eslint/bin/eslint.js",
+                "selector": "source.ts, source.js - meta.attribute-with-value"
+            }
+        }
+    }
+    ```
+
+* To find where a function is defined and used, just hover over the function call or declaration:
+
+  ![Finding a function's definition and usages in Sublime](images/commonEditors/sublimeFunctionDefinitionAndUsage.png)
+
+To learn more, check out the [official](https://www.sublimetext.com/docs/index.html) and [unofficial](https://docs.sublimetext.io/) documentation.
+
+## Atom
+
+If you are new to using Atom, you may find this [getting started video](https://www.youtube.com/watch?v=U5POoGSrtGg) helpful.
+
+To start editing Oppia code:
+
+1. Open the `oppia/` directory (e.g. using CMD+O on MacOS or CTRL+O on Linux or Windows).
+
+2. From the file tree on the left side, select a file to edit. It opens in the center.
+
+   ![Atom with a file open for editing](images/commonEditors/atomEditor.png)
+
+3. To edit the file, just start typing!
+
+Tips:
+
+* Out of the box, Atom is quite a minimal editor that only provides basic functionality. To get more features, install packages. Be warned though that we found the packages of the canonical Atom linting framework, [AtomLinter](https://atomlinter.github.io), to be unmaintained, and we were unable to get them working. We recommend running lint checks from the command line instead.
```

Comment:
Maybe say "... get them working (as of Aug 2021)."

Reply:
done